tests/Server/Middleware/RequireScopeTest.php

```diff
@@ -10,8 +10,6 @@ class RequireScopeTest extends TestCase
     /** @test */
     public function testNoToken()
     {
-        $this->setExpectedException(AccessDeniedHttpException::class);
```

Comment:
Let's update this test to "spy" that the middleware passes!

```php
// We can use $next & $passed as a spy here.
$passed = false;
$next = function () use (&$passed) {
    $passed = true;
};

$request = $this->createRequest(null);

$middleware = new RequireScope(new Token($request, $this->key));
$middleware->handle($request, $next, 'user');

// Since we don't have a token on the request, this should still pass!
$this->assertTrue($passed);
```